src/circuit/gadget/utilities/plonk.rs

```diff
@@ -0,0 +1,348 @@
+use super::{copy, CellValue, UtilitiesInstructions};
+use halo2::{
+    circuit::{Chip, Layouter},
+    plonk::{Advice, Column, ConstraintSystem, Error, Fixed, Permutation},
+    poly::Rotation,
+};
+use pasta_curves::arithmetic::FieldExt;
+use std::marker::PhantomData;
+
+#[allow(clippy::upper_case_acronyms)]
+#[allow(clippy::too_many_arguments)]
+pub trait PLONKInstructions<F: FieldExt>: UtilitiesInstructions<F> {
+    type Var;
```

Comment:
Ditto again: `UtilitiesInstructions` is not used in `PLONKInstructions`.

Reply:
`PLONKInstructions` now uses `UtilitiesInstructions::Var`.